components/rmrk/Gallery/Search/SearchBar.vue

```diff
@@ -504,6 +499,7 @@ export default class SearchBar extends mixins(
           type: 'Search',
           name: this.searchString,
         })
+        this.redirectToGalleryPageIfNeed()
```

Comment:
I am not sure why moving this function `this.redirectToGalleryPageIfNeed()`

Reply:
Because it gets called everytime after you focus and then click anywhere outside, it should only be called in case of enter pressed